packages/javascript/js-client-library/src/types.ts

```diff
@@ -314,7 +314,13 @@ export interface UpdateUserRequest {
     principal: string;
     roles: number[];
     SAMLProviderId?: string;
```

Comment:
Would you mind adding a comment here something along the lines of SAMLProviderId is deprecated but supported to prevent breaking API changes and to use SSOProviderId instead?